server/src/main/scala/de/frosner/broccoli/services/InstanceService.scala

```diff
@@ -25,26 +24,12 @@ class InstanceService @Inject()(nomadClient: NomadClient,
                                 templateService: TemplateService,
                                 nomadService: NomadService,
                                 consulService: ConsulService,
-                                ws: WSClient,
                                 applicationLifecycle: ApplicationLifecycle,
-                                configuration: Configuration)
+                                instanceConfiguration: InstanceConfiguration,
+                                instanceStorage: InstanceStorage)
     extends Logging {
 
-  private lazy val pollingFrequencySecondsString = configuration.getString(conf.POLLING_FREQUENCY_KEY)
-  private lazy val pollingFrequencySecondsTry = pollingFrequencySecondsString match {
-    case Some(string) =>
-      Try(string.toInt).flatMap { int =>
-        if (int >= 1) Success(int) else Failure(new Exception())
-      }
-    case None => Success(conf.POLLING_FREQUENCY_DEFAULT)
-  }
-  if (pollingFrequencySecondsTry.isFailure) {
-    Logger.error(
-      s"Invalid ${conf.POLLING_FREQUENCY_KEY} specified: '${pollingFrequencySecondsString.get}'. Needs to be a positive integer.")
-    System.exit(1)
-  }
-  private lazy val pollingFrequencySeconds = pollingFrequencySecondsTry.get
-  Logger.info(s"Nomad/Consul polling frequency set to $pollingFrequencySeconds seconds")
+  private lazy val pollingFrequencySeconds = instanceConfiguration.pollingFrequency
```

Comment:
The instance service uses the polling frequency, so if at all, the instance service should log it.